app/components/vacancy_form_page_heading_component.rb

```diff
@@ -22,6 +22,14 @@ def caption
     t("jobs.create_job_caption", step: step_process.current_step_group_number, total: step_process.total_step_groups - 1)
   end
 
+  def sub_caption
+    step_process.current_step == :job_role ? "Select all that apply" : nil
+  end
+
+  def sub_caption_class
+    step_process.current_step == :job_role ? "govuk-!-margin-top-4 govuk-!-margin-bottom-4" : nil
+  end
+
```

Comment:
Instead of having this `job_role` logic embedded in the component, could we inject it? That would save the coupling and keep all the `job_role` specific code together in the `job_role.html.slim` page.

EG:
In this file:
``` ruby
def initialize(vacancy, step_process, back_path:, fieldset: true, sub_caption: {})
...
```

In  the `vacancy_form_page_heading_component.html.slim`:

```slim
- if sub_caption.present?
    span.govuk-caption-m class=sub_caption[:class]
      = sub_caption[:text]
```

In `job_role.html.slim`:
```slim
  =  vacancy_form_page_heading(vacancy, step_process, back_path: back_path, fieldset: false, sub_caption: { text: "Select all that apply", class: "govuk-!-margin-top-4 govuk-!-margin-bottom-4")
```